docs/user/security/fips-140-2.asciidoc

```diff
@@ -0,0 +1,63 @@
+[role="xpack"]
+[[xpack-security-fips-140-2]]
+=== FIPS 140-2
+
+The Federal Information Processing Standard (FIPS) Publication 140-2, (FIPS PUB 140-2), 
+titled "Security Requirements for Cryptographic Modules" is a U.S. government computer security standard
+used to approve cryptographic modules.
+
+{kib} offers a FIPS 140-2 compliant mode and as such can run in a Node.js environment configured with an FIPS 
+140-2 compliant OpenSSL3 provider.
+
+[IMPORTANT]
+============================================================================
+The Node bundled with {kib} is not configured for FIPS 140-2. You must configure a FIPS 140-2 compliant OpenSSL3 
```

Comment:
Question for my knowledge: what does `{kib}` render as?

Reply:
It renders as "Kibana" (I have no clue why it has a template). @amyjtechwriter Do you know why we do this?